src/styles/modal.css

```diff
@@ -0,0 +1,74 @@
+/* modal */
+
+.ui.modal {
+  background-color: var(--shade-white);
+  border-radius: 0;
+  box-shadow: none;
```

Comment:
`var(--box-shadow-5);`

Reply:
I don't see the box shadow in Figma. Are you sure it needs to be there?

Reply:
We used it in the component library.